E2E-ORS-Sample-For-Java/src/main/java/objectReplicationMonitor.java

```diff
@@ -0,0 +1,329 @@
+import com.azure.storage.blob.*;
```

Comment:
Try using single imports instead of star imports (like list them individually) There is a setting in intellij that sets the number of imports before it defaults to star - you probably want to increase that by a lot